src/pocketmine/Server.php

```diff
@@ -1653,7 +1653,6 @@ public function __construct(\ClassLoader $autoloader, \ThreadedLogger $logger, $
 		}
 
 		$this->logger->info($this->getLanguage()->translateString("pocketmine.server.networkStart", [$this->getIp() === "" ? "*" : $this->getIp(), $this->getPort()]));
-		define("BOOTUP_RANDOM", @Utils::getRandomBytes(16));
```

Comment:
It's not. It was used in `DebugMemoryCommand` but this was easily changed (and shouldn't affect the command at all).